src/components/ItaliaTheme/Blocks/Listing/CardWithImageTemplate.jsx

```diff
@@ -82,11 +83,9 @@ const CardWithImageTemplate = ({
                             className="img-link"
                           >
                             <figure className="img-wrapper">
-                              <img
+                              <Image
                                 className="listing-image"
-                                src={flattenToAppURL(
-                                  item.image.scales.preview.download,
-                                )}
+                                image={item.image}
```

Comment:
come fa questo componente a sapere che deve essere usata la miniatura 'download'?

Reply:
Crea un srcset con scales[SCALE].download per ogni SCALE in scales